#!/usr/bin/env python
PACKAGE = "simu"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("errorbound", double_t, 0, "errorbound", 0, 0.0, 1.0)
gen.add("X_init", double_t, 0, "X_init", 558640.9252, 555000, 562000)
gen.add("Y_init", double_t, 0, "Y_init", 4196656.6405, 4100000, 4300000)
gen.add("phi_init", double_t, 0, "phi_init", 1.20719921, 0.0, 6.28)
gen.add("steering_init", double_t, 0, "steering", 0, -10, 10)
gen.add("vx_init", double_t, 0, "vx_init", 20, 0.0, 100)
gen.add("Run", bool_t, 0, "start_flag", False, False, True)

exit(gen.generate(PACKAGE, "simu", "DynamicParam"))
